keyhive_wasm/Cargo.toml

```diff
@@ -13,9 +13,9 @@ rust-version.workspace = true
 crate-type = ["cdylib"]
 
 [dependencies]
-from_js_ref = "0.1.0"
-keyhive_core = "0.2.0-alpha.1"
-wasm_refgen = "0.1.0"
+from_js_ref = { path = "../from_js_ref" }
+keyhive_core = { path = "../keyhive_core" }
+wasm_refgen = { path = "../wasm_refgen" }
```

Comment:
This isn't going to work when we try to publish to crates ☹️ Crates doesn't understand paths